dev/com.ibm.ws.cdi.extension.bce_fat/fat/src/com/ibm/ws/cdi/extension/bce_fat/basic/app/beans.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xmlns="https://jakarta.ee/xml/ns/jakartaee"
+xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd"
+version="3.0"
```

Comment:
```suggestion
xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd"
version="4.0"
```
Not sure if this test _needs_ it to be version 4.0 but it would be a good idea to test it anyway